config/deploy.rb

```diff
@@ -0,0 +1,14 @@
+lock '3.2.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.